Rubberduck.Parsing/VBA/ReferenceManagement/IdentifierReferenceResolver.cs

```diff
@@ -160,10 +160,10 @@ private void ResolveLabel(ParserRuleContext context, string label)
             }
         }
 
-        private IEnumerable<IAnnotation> FindIdentifierAnnotations(QualifiedModuleName module, int line)
+        private IEnumerable<ParseTreeAnnotation> FindIdentifierAnnotations(QualifiedModuleName module, int line)
         {
             return _declarationFinder.FindAnnotations(module, line)
-                .Where(annotation => annotation.AnnotationType.HasFlag(AnnotationType.IdentifierAnnotation));
+                .Where(annotation => annotation.Annotation.Target.HasFlag(AnnotationTarget.Identifier));
```

Comment:
That can be integrated into the call to `FindAnnotations` now, right?

Reply:
done 👍